nekoyume/Assets/_Scripts/UI/Widget/Menu.cs

```diff
@@ -244,7 +244,6 @@ public void QuestClick()
             Close();
             var avatarState = States.Instance.CurrentAvatarState;
             Find<WorldMap>().Show(avatarState.worldInformation);
-            Find<HeaderMenuStatic>().UpdateAssets(HeaderMenuStatic.AssetVisibleState.Battle);
```

Comment:
Is this intended?

Reply:
Yes. Because the `WorldMap.Show` changed to call `HeaderMenuStatic.Show`. So duplicate actions were removed.